src/client.c

```diff
@@ -1785,11 +1785,20 @@ void PutClientInServer(void)
 		// qqshka: I found that it sux and added variable which force players spawn ONLY on the base,
 		// so maps like qwq3wcp9 works fine!
 
-		if (isCTF() && ((match_start_time == g_globalvars.time) || cvar("k_ctf_based_spawn")))
+		if (isCTF() && ((match_start_time == g_globalvars.time) || cvar("k_ctf_based_spawn") == 1))
 		{
 			spot = SelectSpawnPoint(
 					streq(getteam(self), "red") ? "info_player_team1" : "info_player_team2");
 		}
+		// Pick between neutral spawn points in the mid of the map and spawn points within home base
+		// to avoid the fish in a barrel problem where flag position is overrun by str/quad/pent and
+		// players are instagibbed over and over again.
+		else if (isCTF() && cvar("k_ctf_based_spawn") == 2)
```

Comment:
please do a small cosmetic change, regarding coding style
`else if (isCTF() && (cvar("k_ctf_based_spawn") == 2))`

Reply:
ops... tihs was not supposed to go here :D